README.md

```diff
@@ -199,11 +199,11 @@ See [here](https://www.javadoc.io/doc/dev.openfeature/sdk/latest/index.html) for
 ## ⭐️ Support the project
 
 - Give this repo a ⭐️!
-- Follow us social media:
+- Follow us on social media:
```

Comment:
Oops. This is wrong in the [template](https://github.com/open-feature/.github/tree/main/templates/READMEs) too.

Reply:
I'll fix the template

Reply:
I did already